packages/server/integrations/JiraTaskIntegrationManager.ts

```diff
@@ -0,0 +1,67 @@
+import makeCreateJiraTaskComment from '../utils/makeCreateJiraTaskComment'
+import JiraProjectId from '~/shared/gqlIds/JiraProjectId'
+import createJiraTask from '../graphql/mutations/helpers/createJiraTask'
+import JiraIssueId from '~/shared/gqlIds/JiraIssueId'
+import {CreateTaskResponse, TaskIntegrationManager} from './TaskIntegrationManagerFactory'
+import {AtlassianAuth} from '../postgres/queries/getAtlassianAuthByUserIdTeamId'
+import {Doc} from '../utils/convertContentStateToADF'
+
+export default class JiraTaskIntegrationManager implements TaskIntegrationManager {
+  public title = 'Jira'
+  public authLoaderKey = 'freshAtlassianAuth' as const
```

Comment:
As discussed, instaed of storing the `authLoaderKey` here the `AtlassianAuth` should be passed in the constructor
```
private auth: AtlassianAuth
constructor(auth: AtlassianAuth) {
  this.auth = auth
}
```
and removed from the methods.

Reply:
Made changes as discussed